charts/backend/Chart.lock

```diff
@@ -7,6 +7,6 @@ dependencies:
   version: 6.14.2
 - name: elasticsearch
   repository: https://helm.elastic.co
-  version: 6.8.13
-digest: sha256:c16420bd8e98a25c44fc5c33061d6d9407971f07bc88c608a8a15863275f9ce8
-generated: "2020-12-22T19:07:54.082297+01:00"
+  version: 6.8.18
```

Comment:
Should this be 6.8.21 / how do I check the hashes.

Reply:
You can check the hash by installing the chart locally. Just bumped the version to 6.8.21 and removed the override as 6.8.21 is now available as chart as well.